### DISCUSSION TITLE
dTao roadmap/timeline

### DISCUSSION THREAD

**Discussion by vstam1** [opentensor/bits#5] [General Bittensor Discussion]
The Rao Testnet testing phase has concluded, and while it's exciting to see progress, I was wondering if there is now a clear roadmap or timeline for the official release of dTao to subtensor mainnet? Given that this upgrade seems to introduce significant changes that could have a substantial impact, it would be helpful to know what to expect in advance.

**Comment by crazydevlegend** [comment]
from what i've heard, later this year, prob in Oct or Nov, but otf has not announced any specific date for it
however, i definitely agree - dtao is going to bring a huge impact for the whole eco

**Reply by DanielGeek** [reply]
Awesome, It will be a great new feature on bittensor, what will be the date that we could see it running?
